app/domain/operations/families/created_or_updated_processor.rb

```diff
@@ -9,6 +9,7 @@ class CreatedOrUpdatedProcessor
       def call(params)
         after_updated_at, family_cv = yield validate(params)
         request_objects = yield create_request_transmittable(family_cv, after_updated_at)
+        _outbound_payload = yield create_outbound_payload(request_objects)
```

Comment:
@jacobkagon What is the reason for not including this in the Family Create operation (app/domain/operations/families/create.rb) as discussed?

* We will hit the database multiple times using the current approach.